examples/jsm/loaders/FBXLoader.js

```diff
@@ -169,8 +169,17 @@ class FBXLoader extends Loader {
 		}
 
 		// console.log( fbxTree );
+		let textureLoader = this.manager.getHandler( url );
```

Comment:
It seems `url` is not defined at this point.

Reply:
How about implementing the change in `loadTexture()` instead?

Reply:
Thank you.
Since the loadTexture method appears to be designed for configuring explicit loaders based on file extensions, adding a TextureLoader check here would cause side effects to users who are already using addHandler.

Reply:
@Mugen87 
I understand, the extension is not clear at this point.
So loadTexture is needed.
https://github.com/mrdoob/three.js/pull/31032/commits/d79a1e16b797dcbd17332212af7732d1a6f830be

Reply:
That looks better. However, the below `if ( ! loader ) {` block is obsolete now since the loader can't be `null` at that point.

I would like to test the PR but I've realize we don't have a single textured FBX asset in the repository. Do you mind sharing one so it's possible to verify the change?

Reply:
[cube.zip](https://github.com/user-attachments/files/20031891/cube.zip)
here is cube fbx with brick texture.

Reply:
[cube_embed.zip](https://github.com/user-attachments/files/20031898/cube_embed.zip)
I’ve attached embed fbx as well for your reference.

Reply:
Okay, looking good. 

Please remove the code section mentioned in https://github.com/mrdoob/three.js/pull/31032#discussion_r2072561965 though.

Reply:
Never mind! I've done it by myself: 80ddfbca3c3879c503f93f97292f4207ed6ee6a0